apps/condo/domains/contact/schema/Contact.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Generated by `createschema contact.Contact 'property:Relationship:Property:SET_NULL; name:Text; phone:Text; unitName?:Text; email?:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/contact/access/Contact')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+
+
+const Contact = new GQLListSchema('Contact', {
+    schemaDoc: 'Contact information of a person. Currently it will be related to a ticket, but in the future, it will be associated with more things',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        property: {
```

Comment:
organization

Reply:
Please, answer to my [question](https://github.com/open-condo-software/condo/pull/154#discussion_r648031591), that I initially wrote.
Copied:
> It will reduce query complexity, but will increase code complexity, because we will need to add validation for connected `Property` and `Organization` to be associated with each other, to prevent cases, where connected `Property` is belonging to another connected `Organization`.
Is this critical currently?

Reply:
Added in `fd8834f3`.